lib/src/screens/contact/contact_list_page.dart

```diff
@@ -120,6 +120,7 @@ class ContactListPage extends BasePage {
         if (isCopied) {
           await Clipboard.setData(ClipboardData(text: contact.address));
           await showBar<void>(context, S.of(context).copied_to_clipboard);
+          Navigator.of(context).pop(contact);
```

Comment:
this will pop the screen ok, but we need it to behave as the normal flow (i.e take the address from the selected contact and use it in the address field